### PR TITLE
E1507 Refactoring Review Files Controller & Submitted Content Controller

### PR DESCRIPTION
**Pull request by KeleiAzz**
fixed bug #485, and refactored review files controller.

### CODE REVIEW

File: app/controllers/auth_controller.rb [hunk 1]
```diff
@@ -20,13 +20,13 @@ def login
     else
       user = User.find_by_login(params[:login][:name])
 
-      if user and user.valid_password?(params[:login][:password])
```
Comment (yangsong8) [inline app/controllers/auth_controller.rb]: This is the code to enable password verification, please put it back.

File: app/models/assignment_participant.rb [hunk 2]
```diff
@@ -84,33 +84,32 @@ def includes?(participant)
...
-                                        :reviewed_object_id => assignment.id)
+    ParticipantReviewResponseMap.create(reviewee_id: self.id, reviewer_id: reviewer.id,
+                                        reviewed_object_id: assignment.id)
   end
 
   def assign_quiz(contributor,reviewer,topic)
-    participant_id=AssignmentParticipant.where(topic_id: topic, parent_id:  contributor.parent_id).first.id
+    participant_id=AssignmentParticipant.where(topic_id: topic, parent_id: contributor.parent_id).first.id
 
-    quiz = QuizQuestionnaire.find_by_instructor_id(contributor.id)
```
Comment (yangsong8) [inline app/models/assignment_participant.rb]: Those changes are actually good to have, but are they in your project scope?

Comment (KeleiAzz) [inline app/models/assignment_participant.rb]: I changed the .find_by_instructor_id to .where(instructor_id:) according to the global rules.

**Comment by yangsong8** [comment]
By the way I cannot find if your pull request passes Travis CI or not. Do you have any idea on how to get it?

**Comment by yangsong8** [comment]
Hi guys, we are planning to merge your code to the rails 4 branch. Before we do this, there are a couple of minor fixes, it would be great if you can do:

- [ ] When try to rename a file, the pop-up says "specify name for undefined". "Undefined" should be the file name.
- [ ] Logger statement in student_review/list should be removed.
- [ ] You did not change any DB design, right? (please check it if your answer is "no")
- [ ] Your project passes Travis CI, right? (please check it if your answer is "yes")

We will merge your project after those are all checked.

Thanks,

Yang

**Comment by KeleiAzz** [comment]
Seems the Travis CI build passed now.

**Pull request closed without merging**